### PR TITLE
feat(Snackbar):Adding snackbar component into component library

### PR DESCRIPTION
**Pull request by Chirag-kumar-dev**
### Description
Adding snackbar component .

### CODE REVIEW

**Comment by rahul31agrawal** [comment]
Looks good to me, 
Optional suggestion:- if length of the snack bar will be less, It will look better.

File: src/Style/common.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .code{
-    width: 500px;
```
Comment (rahul31agrawal) [inline src/Style/common.css]: try to use rems instead of px, it will be more responsive,
you can use px to rem converter if you need any specific value.

**Review by rahul31agrawal — APPROVED** [review]

File: src/Style/common.css (hunk 1 above)
Comment (Chirag-kumar-dev) [reply src/Style/common.css]: Thanks i change this

**Comment by Chirag-kumar-dev** [comment]
> Looks good to me, Optional suggestion:- if length of the snack bar will be less, It will look better.

yes i change this,Thanks

**Review by sohan9819 — APPROVED** [review]
LGTM

**Comment by psychoactiv** [comment]
Looks good to merge

**Review by psychoactiv — APPROVED** [review]

**Pull request merged**